test/command_line/suite/grndb/test_check.rb

```diff
@@ -32,26 +32,46 @@ def test_normal
     end
   end
 
+  def test_normal_info_log
+    groonga("table_create", "info", "TABLE_NO_KEY")
+    grndb("check", "--log-level", "info")
+    messages = <<-MESSAGE
+Checking database: <#{@database_path}>
+Database is not locked: <#{@database_path}>
+Database is not corrupted: <#{@database_path}>
+Database is not dirty: <#{@database_path}>
+[info] object is not locked
+[info] object is not corrupted
+    MESSAGE
+    messages.split("\n").each do |message|
+      assert_includes(File.read(@log_path), message)
```

Comment:
We should not use `assert_includes`. It doesn't show enough debug information when the assertion is failed. For example, we can't know what is logged and what isn't logged when the assertion is failed.

We should normalize log content and use `assert_equal` instead:

```ruby
groonga("table_create", "info", "TABLE_NO_KEY")
FileUtils.rm(@log_path)
grndb("check", "--log-level", "info")
assert_equal(<<-MESSAGE, normalize_log(File.read(@log_path)))
1970-01-01 00:00:00.000000|i| Checking database: <#{@database_path}>
1970-01-01 00:00:00.000000|i| Database is not locked: <#{@database_path}>
1970-01-01 00:00:00.000000|i| Database is not corrupted: <#{@database_path}>
1970-01-01 00:00:00.000000|i| Database is not dirty: <#{@database_path}>
1970-01-01 00:00:00.000000|i| [info] object is not locked
1970-01-01 00:00:00.000000|i| [info] object is not corrupted
MESSAGE
```

Reply:
I've got it.  fixed.